drawee/src/main/java/com/facebook/drawee/generic/GenericDraweeHierarchy.java

```diff
@@ -421,13 +423,6 @@ public void setPlaceholderImage(Drawable drawable, ScaleType scaleType) {
     getScaleTypeDrawableAtIndex(PLACEHOLDER_IMAGE_INDEX).setScaleType(scaleType);
   }
 
-  /**
-   * @return true if there is a placeholder image set.
-   */
-  public boolean hasPlaceholderImage() {
-    return getParentDrawableAtIndex(PLACEHOLDER_IMAGE_INDEX) != null;
-  }
-
```

Comment:
I meant that an app using Fresco might be calling the method. In fact we do have a caller to this in our codebase.

You're right that it doesn't behave as expected but I'll deal with that.